test/unit/player.test.js

```diff
@@ -1601,14 +1601,22 @@ QUnit.test('src_ does not call loadTech is name is titleCaseEquals', function(as
         tech: 'html5'
       };
     },
+    options_: {},
+    tech_: {
+      constructor: {
+        prototype: {}
+      }
+    },
+    techCall_() {},
     techName_: 'Html5',
-    ready() {},
+    // ready() {},
+    load() {},
     loadTech_() {
       loadTechCalled++;
     }
   };
 
-  Player.prototype.src_.call(playerProxy);
+  Player.prototype.src_.call(playerProxy, {});
```

Comment:
The changes to this test were necessary because `src_()` no longer calls `ready()`. Instead, it immediately loads the source into the tech because the only cases where this is possible is if the tech did not change; so, it should already be ready.